tests/utils.py

```diff
@@ -68,3 +71,12 @@ def create_rohmu_config(tmpdir, *, compression=True, encryption=True):
             }
         )
     return RohmuConfig.parse_obj(config)
+
+
+def get_clickhouse_version(command: List[Union[str, Path]]) -> tuple[int]:
```

Comment:
`tuple[int]` is not a tuple of ints, it's a tuple with a single int, you need `tuple[int, ...]` or `Sequence[int]` (I think the second one is more clear, it requires less knowledge to understand what it means).

Reply:
You're right, somehow I thought it type-checked any homogenous tuple type.